src/main/java/ladder/step2/domain/LadderLine.java

```diff
@@ -0,0 +1,31 @@
+package ladder.step2.domain;
+
+import ladder.step2.domain.strategy.LadderLineStrategy;
+
+import java.util.List;
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
+
+import static java.util.stream.Collectors.collectingAndThen;
+import static java.util.stream.Collectors.toList;
+
+public class LadderLine {
+  private final List<Boolean> lines;
+
+  private LadderLine (List<Boolean> lines) {
+    this.lines = lines;
+  }
+
+  public static LadderLine of (int countOfPerson, LadderLineStrategy strategy) {
+    final boolean[] prev = { false };
```

Comment:
다음과 같이 초기화 하는 방법도 있을 것 같아요 :)
```suggestion
    final boolean[] prev = new boolean[countOfPerson];
```

Reply:
이렇게 초기화 하는 의도가 무엇인가요?

Reply:
현재 명시적으로 false를 초기값을 주고 계시고 있는 방법 외에
boolean 배열은 자동으로 false로 초기화가 되는 방법도 있다는 것을 참고하기 위해 리뷰 드렸습니다 :)